.github/workflows/publish.yml

```diff
@@ -20,14 +23,14 @@ jobs:
       - name: Test
         run: yarn test --full --ci
         env:
-          SOURCECRED_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SOURCECRED_GITHUB_TOKEN: ${{ secrets.SOURCECRED_GITHUB_TOKEN }}
```

Comment:
Why this change? If we want the Credbot token for its permissions, we could use `CREDBOT_TOKEN`. But since this is for `yarn test`, the default GITHUB_TOKEN should be fine, no?

Reply:
Adding `persist-credentials: false` disables the default GITHUB_TOKEN that goes into Actions, so changed it to use this one instead.

Reply:
Got it.